package.json

```diff
@@ -10,7 +10,7 @@
     "format": "prettier -w . --experimental-ternaries",
     "start": "yarn package && node dist/index.cjs",
     "package:client": "rimraf out/ && yarn build && mkdir -p out/ && cp -r index.html dist/ assets/ out/",
-    "package:server": "ncc build -m server.cjs -o dist/ --target es2022 --source-map",
+    "package:server": "tsc --module commonjs supervisor-helper.ts --outdir out --esmoduleinterop true --declaration && mv out/supervisor-helper.js out/supervisor-helper.cjs && ncc build -m server.cjs -o dist/ --target es2022 --source-map",
```

Comment:
It would be great to unify and not have to transpile individual files, but we are blocked by the migration of the main server script to typescript. There is some wip on https://github.com/gitpod-io/xterm-web-ide/tree/ft/ts-migration, but the state on the branch is not functional.